### PR TITLE
NOC-244: fix excessive scroll

### PR DESCRIPTION
**Pull request by ketewan**
## Change Summary

Some adjustments to get rid of text input overflow and excessive scrolls in filters pop-up

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [x] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

File: packages/nc-gui/components/project/spreadsheet/components/ColumnFilter.vue [hunk 1]
```diff
@@ -489,7 +489,7 @@ export default {
 }
 
 .v-input {
-  min-width: 100%;
+  min-width: 0;
```
Comment (Faradey27) [inline packages/nc-gui/components/project/spreadsheet/components/ColumnFilter.vue:492]: can this break smth?

File: packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue [hunk 2]
```diff
@@ -14,10 +14,10 @@
     @click.stop
     @change="$emit('change')"
   >
-    <template #selection="{item}">
-      <v-icon small class="mr-1">
-        {{ item.icon }}
-      </v-icon> {{ item.title }}
+    <template #prepend-inner>
```
Comment (Faradey27) [inline packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:17]: just curious what does prepend-inner do? what was #selection doing?(I thought that this is the way to pass data)

File: packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue [hunk 3]
```diff
@@ -14,10 +14,10 @@
...
+      <v-icon small class="mt-1">
```
Comment (Faradey27) [inline packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:18]: why we do not need mr anymore?

File: packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue (hunk 2 above)
Comment (ketewan) [reply packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:17]: These are slots - Vue way to pass template content (https://vuejs.org/guide/components/slots.html)
You can override different default slots of a component, #selection was for the whole input, and #prepend-inner is a special slot to add only item before input (like icon). You can see the whole list of slots [here](https://vuetifyjs.com/en/api/v-autocomplete/#slots)
It is also possible to do similar thing [prepend-inner-icon](https://vuetifyjs.com/en/api/v-autocomplete/#props-prepend-inner-icon) prop, but it is not possible to change icon size while using a prop

Comment (ketewan) [reply packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:17]: The override for #selection didn't work well, either adding overflow for inner text or overriding placeholder-text with input/ I decided to rely on default behavior and add icon with a special slot for it

File: packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue (hunk 3 above)
Comment (ketewan) [reply packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:18]: `prepend-inner` already has `mr`

File: packages/nc-gui/components/project/spreadsheet/components/ColumnFilter.vue (hunk 1 above)
Comment (ketewan) [reply packages/nc-gui/components/project/spreadsheet/components/ColumnFilter.vue:492]: It was already broken, hope it won't break it worse :DD

File: packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue (hunk 2 above)
Comment (Faradey27) [reply packages/nc-gui/components/project/spreadsheet/components/FieldListAutoCompleteDropdown.vue:17]: awesome! thank you!

**Review by Faradey27 — APPROVED** [review]
By the way, do we have dev deploy to test PR without local run and local database generation?

**Pull request merged**